Gemfile

```diff
@@ -65,7 +65,7 @@ group :development do
   gem 'pry-remote'
 
   # testing against the client
-  gem 'routemaster-client'
+  gem 'routemaster-client', git: 'https://github.com/deliveroo/routemaster-client.git', ref: 'b29fc75'
```

Comment:
Since deliveroo/routemaster-client#5 is merged I believe we don't need this anymore